Common/Utilities.cs

```diff
@@ -2804,14 +2793,10 @@ public static void CreateCertificate(string domainName, string pfxName, string c
             }
             else
             {
-                //File.Copy(
-                //    Path.Combine(Utilities.ProjectPath, "Asset", "SampleTestCertificate.pfx"),
-                //    Path.Combine(Utilities.ProjectPath, "Asset", pfxName),
-                //    overwrite: true);
             }
         }
 
-        public static void UploadFileToWebApp(IPublishingProfile profile, string filePath, string fileName = null)
+        public static async Task UploadFileToWebApp(IPublishingProfile profile, string filePath, string fileName = null)

```

Comment:
Nit: best practice to end async method with the "Async" prefix as we have done in our SDK.

Reply:
Resolved